source/atlas-cli-deploy-docker.txt

```diff
@@ -309,7 +273,89 @@ of Docker Compose.
 
       .. code-block:: sh
 
-         mongosh mongodb://root:{your-password}@localhost:27017/?directConnection=true
+         mongosh "mongodb://user:pass@localhost:27019/?directConnection=true"
+
+Generate a List of Dependencies
+-------------------------------
+
+You can generate a list of the dependances for the 
+``mongodb/mongodb-atlas-local`` Docker image.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install syft.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install syft
+
+      To learn more, see the :github:`syft README 
+      </anchore/syft/blob/main/README.md>`.
+
+.. step:: Run syft.
+
+   .. code-block:: sh
+
+      syft mongodb/mongodb-atlas-local
+
+Verify the Image Signature
+--------------------------
+
+You can verify the signature of the ``mongodb/mongodb-atlas-local`` 
+Docker image.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install cosign.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install cosign
+
+      To learn more, see the `cosign Installation 
+      <https://docs.sigstore.dev/system_config/installation/>`__
```

Comment:
Same here: it is strange to see this after the fact. Instead, I would do this:
You can verify the signature .... using Cosign. To learn more, see ... link to Cosign.

A suggestion: should we move these long http links to the snooty.toml and use shorter replacements in text?